src/src/main/java/service/Impl/ProductServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package service.Impl;
+
+import model.Product;
+import service.DataBase;
+import service.ProductService;
+
+import java.util.List;
+
+public class ProductServiceImpl implements ProductService {
+    @Override
```

Comment:
we need to create new instance here:
 private ProductDao  productDao = new ProductDaoFactory.getInstance();

Reply:
all fixed